meowmore.sh

```diff
@@ -0,0 +1,4 @@
+!#/bin/bash
+
+# MWAHAHAHAHA THE POWER!!!!
+while true; do xmessage Hi -buttons GET\ OUT; done;
```

Comment:
Unfortunately, this is not meant to be a graphical program, so this cannot be merged.